### PR TITLE
(feat) 03 2572 Implement the new snack bar in patient management(part 1)

### PR DESCRIPTION
**Pull request by NYESIGA**
Use the new snackbar in all places where Toast and Notification were used in patient management in first 3 apps of patient management that is active-visit-app,appointment-app and patient-list-management-app   

 Screenshots

Screenshot (95)

https://openmrs.atlassian.net/browse/O3-2572

### CODE REVIEW

File: packages/esm-patient-list-management-app/src/list-details-table/list-details-table.component.tsx [hunk 1]
```diff
@@ -215,17 +215,17 @@ const ListDetailsTable: React.FC<ListDetailsTableProps> = ({
       mutateListMembers();
       mutateListDetails();
 
-      showToast({
-        critical: true,
+      showSnackbar({
+        isLowContrast: false,
```
Comment (hadijahkyampeire) [inline packages/esm-patient-list-management-app/src/list-details-table/list-details-table.component.tsx:219]:
```suggestion
        isLowContrast: false,
        isLowContrast: true
```

**Review by hadijahkyampeire — CHANGES_REQUESTED** [review]
Thanks @NYESIGA just a few improvements, mainly about the isLowContrast prop, just a note; All success snackbars should have a true lowcontrast prop and error snackbars should have false but I think if you don't pass the prop it will default to false, just ensure to test error snackbars and confirm that they all have a high contrast i.e black background.

**Review by NYESIGA — COMMENTED** [review]
@hadijahkyampeire  i have made changes as you suggested

File: packages/esm-patient-list-management-app/src/add-patient/add-patient.component.tsx [hunk 2]
```diff
@@ -62,17 +62,17 @@ const AddPatient: React.FC<AddPatientProps> = ({ closeModal, patientUuid }) => {
       patientList
         .addPatient()
         .then(() =>
-          showToast({
+          showSnackbar({
```
[the remark this answers is not in the document]
Comment (hadijahkyampeire) [reply packages/esm-patient-list-management-app/src/add-patient/add-patient.component.tsx:65]: @NYESIGA I still don't see isLowContrast

File: packages/esm-patient-list-management-app/src/create-edit-patient-list/create-edit-list.component.tsx [hunk 3]
```diff
@@ -47,9 +47,9 @@ const CreateEditPatientList: React.FC<CreateEditPatientListProps> = ({
     if (isEditing) {
       editPatientList(patientListDetails.uuid, cohortDetails)
         .then(() => {
-          showToast({
+          showSnackbar({
```
[the remark this answers is not in the document]
Comment (hadijahkyampeire) [reply packages/esm-patient-list-management-app/src/create-edit-patient-list/create-edit-list.component.tsx:50]: @NYESIGA you missed this too

File: packages/esm-patient-list-management-app/src/list-details/list-details.component.tsx [hunk 4]
```diff
@@ -88,18 +88,18 @@ const ListDetails = () => {
   const confirmDeletePatientList = useCallback(() => {
     deletePatientList(patientListUuid)
       .then(() => {
-        showToast({
+        showSnackbar({
```
[the remark this answers is not in the document]
Comment (hadijahkyampeire) [reply packages/esm-patient-list-management-app/src/list-details/list-details.component.tsx:91]: Also this @NYESIGA

**Review by hadijahkyampeire — CHANGES_REQUESTED** [review]
Still a few success snackbars missing isLowContrast, please ensure all success snackbars must have `isLowContrast: true`

**Review by NYESIGA — COMMENTED** [review]
i have removed   this console.error('Error updating list: ', error); on line 62

**Review by NYESIGA — COMMENTED** [review]
@hadijahkyampeire is there any change i need to make

**Comment by hadijahkyampeire** [comment]
@NYESIGA let me know if you wanna pair on the lint issue.

**Review by njiddasalifu — CHANGES_REQUESTED** [review]
@NYESIGA I think you are getting the fail due to many commits. Can you squash them into one commit and try again.

**Comment by hadijahkyampeire** [comment]
@NYESIGA I am gonna close this since I can push to it and open a new one.

**Pull request closed without merging**